storybookjs/storybook
# Core: Fix possible "write after end" exception for response stream

Issue: #13006 

## What I did

I removed the code that closes the response to keep the "buffer / stream" open, making the hot-reload work again.

![Screenshot from 2020-11-03 23-00-07](https://user-images.githubusercontent.com/14007153/98060567-cadb2880-1e41-11eb-87ae-f9188e4f477a.png)

> I'm not sure if it is the best solution. Feel free to propose changes.

## How to test

1. Pull this branch and run `yarn bootstrap`
2. Go to `examples/react-ts` and run `yarn storybook`
3. Make any changes to the `examples/react-ts/src/button.tsx` component and save the file
4. See the hot-reload working again.
